plugins/statistics.koplugin/main.lua

```diff
@@ -188,7 +193,7 @@ function ReaderStatistics:isEnabledAndNotFrozen()
 end
 
 -- Reset the (volatile) stats on page count changes (e.g., after a font size update)
-function ReaderStatistics:onDocumentRerendered()
+function ReaderStatistics:onDocumentRerendered(use_pagemap_for_stats_updated)
```

Comment:
I find this new argument and your `self:onDocumentRerendered(true)` below a bit odd:
Nothing else currently pass an argument, but if we later want to pass one (and may be a more serious one, really more on-topic with DocumentRerendered than "pagemap toggled" is), we will conflict.
The one you pass here is really pertinent to this plugin only, so it feels we may better use another way to let this fact known to this specific `ReaderStatistics:onDocumentRerendered()` handler here.